clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesApiConverter.groovy

```diff
@@ -240,6 +240,23 @@ class KubernetesApiConverter {
     return handlerBuilder.build()
   }
 
+  static Container toInitContainer(KubernetesInitContainerDescription initContainer) {
+    return toContainer(new KubernetesContainerDescription(
```

Comment:
Is an init container always a strict subset of a regular container in terms of the properties it supports?

Reply:
It is. See `initContainers` section [here](https://kubernetes.io/docs/api-reference/v1.8/#podspec-v1-core).